album_art.py

```diff
@@ -0,0 +1,84 @@
+import requests
+import os
+import string
+import json
+from bs4 import BeautifulSoup
+
+def fetch(song, **kwargs):
+    if kwargs['backend'] == 'lastfm':
+        image = fetch_from_lastfm(song)
+    elif kwargs['backend'] == 'google':
+        image = fetch_from_google(song)
+    return image
+
```

Comment:
There are pep8 plugins for almost all popular editors or you can install the `pep8` apt package and use it like `pep8 <your file name>` to check if your file conforms to pep8 guidelines.

Only acceptable error is `E501`